News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/model/OcsUser.java

```diff
@@ -35,6 +39,14 @@ public void setDisplayName(String displayName) {
         this.displayName = displayName;
     }
 
+	public @Nullable String getAvatarUrl(@Nullable String ownCloudRootPath) {
```

Comment:
I hope annotations are fine. They will make porting the code to kotlin much easier if desired.